CMakeLists.txt

```diff
@@ -1,186 +1,11 @@
-cmake_minimum_required (VERSION 2.6)
-project(ale)
-set(ALEVERSION "0.6.0")
+cmake_minimum_required (VERSION 3.14)
 
+project(ale VERSION 0.6.1
```

Comment:
Is this version number picked up by version.hpp.in?

Reply:
Yes, `configure_file ("version.hpp.in" "version.hpp")` in CMakeLists provides the `@PROJECT_VERSION@` replacement.